app/services/transformer_loader.rb

```diff
@@ -50,20 +50,36 @@ def solr
     @solr ||= SolrService.new
   end
 
-  def solr_doc_for(dataset_record:)
+  def solr_doc_for(dataset_record:) # rubocop:disable Metrics/AbcSize
     Honeybadger.context(dataset_record_id: dataset_record.id, provider: dataset_record.provider,
                         dataset_id: dataset_record.dataset_id)
     metadata = mapper.call(source: dataset_record.source)
-    SolrMapper.call(
-      metadata:,
-      dataset_record_id: dataset_record.id,
-      dataset_record_set_id: dataset_record_set.id
-    )
+    check_mapping_success(dataset_record:)
```

Comment:
this would be a little more intuitive to me if we moved its guard condition up into this line. so something like:

```
report_mapping_success(dataset_record:) if ignore?(dataset_record:)
```

and rename the method, instead of having the check be in the method itself.